l1-contracts/test/staking/deposit.t.sol

```diff
@@ -50,17 +57,26 @@ contract DepositTest is StakingBase {
     // it reverts
 
     staking.deposit({_attester: ATTESTER, _withdrawer: WITHDRAWER, _onCanonical: true});
+    staking.flushEntryQueue();
 
     stakingAsset.mint(address(this), DEPOSIT_AMOUNT);
     stakingAsset.approve(address(staking), type(uint256).max);
 
-    address magicAddress = address(staking.getGSE().CANONICAL_MAGIC_ADDRESS());
+    bytes32 stakingSlot = keccak256("aztec.core.staking.storage");
+    bytes32 flushableEpochSlot = bytes32(uint256(stakingSlot) + 7);
+    vm.store(address(staking), flushableEpochSlot, bytes32(uint256(0)));
```

Comment:
What is going on here? Why are you doing it like this? 

When you are importing the `stdStorage` (which I am not sure why you did here?) you should be able to use the `checked_write` to update it. Would prefer that to not run into issue with hardcoded values here 🤷. If you cannot do that directly because there are no getters, it might be that we should really have those getters, such that it is also easy to figure out if you can actually call it or not without needing to call and revert 👀.


Example from multi proof tests
```solidity
    stdstore.enable_packed_slots().target(address(rollup)).sig("getActivityScore(address)").depth(1)
      .with_key(alice).checked_write(maxActivityScore);
```